README.md

```diff
@@ -24,13 +24,31 @@ directly without another `pip install`.
 
 - Create an OAuth application on CommCare HQ using Django Admin at the URL
   `<hq_host>/admin/oauth2_provider/application/` with the following settings:
+
   - Redirect URIs: `<superset_host>/oauth-authorized/commcare`
+
   - Leave "Post logout redirect URIs" empty.
+
   - Client type: Confidential
+
   - Authorization grant type: Authorization code
-  - Give your OAuth application a name, like "CommCare Analytics" or "HQ Superset"
+
+  - Give your OAuth application a name, like "CommCare Analytics" or "HQ
+    Superset". This name will appear in CommCare HQ's dialog box
+    requesting authorization from the user. e.g.
+
+    > **Authorize *CommCare Analytics* ?**
```

Comment:
nit: not sure putting the content of the popup here is helpful.

Reply:
e013791